app/components/CoinListCell/CoinListCell.js

```diff
@@ -12,73 +16,100 @@ export default class CoinListCell extends Component {
   }
 
   handleCellPress = () => {
-    LayoutAnimation.easeInEaseOut();
-    this.setState(
-      {
-        isExpanded: !this.state.isExpanded
-      }
-    )
+    LayoutAnimation.easeInEaseOut()
+    this.setState({
+      isExpanded: !this.state.isExpanded,
+    })
   }
 
   render() {
+    const s = coinListCellStyles
```

Comment:
Let's change `s` into `styles`. Easier to read and understand.

Reply:
I specifically named it a one letter variable because it just makes readability easier with long names. I feel like having `<Component style={styles.longNameOfStyle}/>` is redundant rather than just doing `<Component style={s.longNameOfStyle}/>`

Reply:
Yea but then the variable name is not explicit and harder to reader because you need to reference it from above and see what going on. Your call. I would prefer more verbose variables.

Reply:
Given that there is no ambiguity on what the `s` stands for and this is a small private project where both of us can agree `s` means `styles` it can become a convention in the code base.

The benefit of not having the code break into multiple lines because is a bigger positive than explicitly writing it out imo. So I think we should keep it

Reply:
Cool cool